sources/common/infoschema.go

```diff
@@ -191,6 +192,7 @@ func (is *InfoSchemaImpl) processTable(conv *internal.Conv, table SchemaAndName,
 		return t, fmt.Errorf("couldn't get foreign key constraints for table %s.%s: %s", table.Schema, table.Name, err)
 	}
 
+	table.Id = internal.GenerateTableId()
```

Comment:
Where was this Id generated earlier?

Reply:
Sorry I had to assign the value not generate it. It is generated a few line before this only in the same function